changelog.go

```diff
@@ -123,3 +133,31 @@ func GenerateChangeLog(c *Config, p *Project) ([]byte, error) {
 
 	return out.Bytes(), nil
 }
+
+// Extract and display contributors from release body
+func processContributors(out *bytes.Buffer, usernames []string, p *Project, userExists map[string]bool) {
+	var contributors []string
+	for _, username := range usernames {
+		if _, checked := userExists[username]; !checked {
+			// Verify user exists to avoid 404 on image load
+			_, resp, err := p.GitHub.api.Users.Get(context.TODO(), username)
+			userExists[username] = err == nil && resp.StatusCode == http.StatusOK
```

Comment:
Instead of using GitHub API, can we send HEAD request to `https://wsrv.nl/?url=https://github.com/{username}.png` and check the response is not 404?

I'd like to suggest this change because

- API has rate limit and the rate limit without API token is very small
- We no longer need to rely on `GitHub` instance so `Project` struct doesn't need to hold `GitHub` instance

Reply:
Good call, fixed.